contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -62,23 +61,21 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     // The first key is the blockchain ID of the destination chain, the second key is the message ID, and the value is the info
     // for the uniquely identified message.
     mapping(
-        bytes32 destinationBlockchainID => mapping(uint256 messageID => SentMessageInfo messageInfo)
+        bytes32 destinationBlockchainID => mapping(bytes32 messageID => SentMessageInfo messageInfo)
```

Comment:
I think we can actually get rid of the first key, since `messageID` is now unique regardless of which `destinationBlockchainID` it's sending to, and is unique to `messageNonce`